sources/osgShadow/ShadowTexture.js

```diff
@@ -77,8 +77,9 @@ define( [
 
             }.bind( this ) );
 
-
+            // tells shader where to find it
```

Comment:
Ok we should probably talk about this. I will spend more time to understand this part.